db/migrate/20201201184505_add_member_timestamps_to_users.rb

```diff
@@ -0,0 +1,10 @@
+class AddMemberTimestampsToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :company_member_since, :date
+    add_column :users, :company_member_until, :date
+
+    User.where(company_member: true).find_each do |user|
+      user.update(company_member_since: Date.parse('1960-10-30'))
+    end
```

Comment:
I can add up and down methods yes, but the thing is, that logic must run between migrations because it uses a field that is removed on the next migration.

Reply:
Yeah, absolutely, I know why you did it here. The way to avoid this is to separate the migrations in different PRs, so that you can create the new column(s) (first PR), backfill them (along with the first one) and remove the old column (next PR).
I know doing this is disrupting the work you have already done, so feel free to change it or to leave like this. It's up to you.